_tutorials/104_rests/step-00/104_rests_step-00-desc.html

```diff
@@ -1,7 +1,9 @@
 <div>
     <p>
-        In this tutorial, you will learn how to encode different types
-        of rests from the Common Music Notation repertoire using MEI.
+        In this tutorial, you will learn how to encode different types of rests from the Common Music Notation repertoire using MEI.
+    </p>
+    <p>
+        Please consult <a href="/tutorials/100-structure.html" target="_blank" rel="noopener, noreferrer">XML basics and minimal MEI structure</a> and/or the <a href="/tutorials/101-quickstart.html" arget="_blank" rel="noopener, noreferrer">Quickstart</a> tutorial if you need more information to complete the task below.
```

Comment:
```suggestion
        Please consult <a href="/tutorials/100-structure.html" target="_blank" rel="noopener, noreferrer">XML basics and minimal MEI structure</a> and/or the <a href="/tutorials/101-quickstart.html" target="_blank" rel="noopener, noreferrer">Quickstart</a> tutorial if you need more information to complete the task below.
```

Reply:
@mfeustle You can add the suggested change (if you accept it) directly to this PR by clicking the "Commit suggestion" button above.